Doc/whatsnew/3.11.rst

```diff
@@ -99,6 +99,17 @@ Optimizations
   almost eliminated when no exception is raised.
   (Contributed by Mark Shannon in :issue:`40222`.)
 
+* Method calls with keywords are now up to 20% faster due to bytecode
+  changes which avoid creating bound method instances. Previously, this
+  optimization was applied only to method calls with purely positional
+  arguments.
+  (Contributed by Ken Jin and Mark Shannon in :issue:`26110`, based on ideas
```

Comment:
Is the issue correct? In https://bugs.python.org/issue26110 I don't see any PR for this

Reply:
Yes, it didn't appear for some reason, there was some recent discussion about other issues having the same problem https://mail.python.org/archives/list/python-committers@python.org/thread/M2Z3WSJ4SGGH2GQ7D5NELAZSD3PECG7Q/ .